package.json

```diff
@@ -18,17 +26,20 @@
     "marked": "^0.4.0",
     "prettier": "^2.8.4",
     "ts-node": "^8.4.1",
+    "tsup": "^7.2.0",
     "typedoc": "^0.11.1",
     "typescript": "^5.0.4",
     "vitest": "^0.28.4"
   },
   "scripts": {
     "clean": "rm -rf ./dist && mkdir dist",
     "test": "vitest run --coverage && yarn run compile",
+    "tsup:test": "vitest run --coverage && yarn run tsup:watch",
     "test:ci": "vitest run",
     "compile": "tsc",
     "compile:dist": "tsc --project tsconfig.dist.json",
-    "build": "yarn run clean && yarn run build:es && yarn run build:cjs",
+    "tsup:watch": "tsup --watch",
+    "build": "tsup",
     "build:es": "tsc --project tsconfig.build.json --module esnext --outDir \"./dist/es\"",
     "build:cjs": "tsc --project tsconfig.build.json --module commonjs --outDir \"./dist/commonjs\"",
```

Comment:
question: are those two build scripts still needed ?

Reply:
No. I left them there in case you didn't merge the pr. I'll be happy to remove them for you

Reply:
> yeah please remove them 👍

Done!